pkg/controller/job/pod_failure_policy_test.go

```diff
@@ -0,0 +1,471 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package job
+
+import (
+	"testing"
+
+	batch "k8s.io/api/batch/v1"
```

Comment:
Nit: `batchv1` to make it explicit

Reply:
See the response here: https://github.com/kubernetes/kubernetes/pull/111113#discussion_r936304906